resources/carrier/zh_Hant/86.txt

```diff
@@ -0,0 +1,64 @@
+# Copyright (C) 2016 The Libphonenumber Authors
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Translations of en/86.txt from each carrier's wikipedia page and in
+# some cases the carrier's own home page.
+
+86130|中國聯通
+86131|中國聯通
+86132|中國聯通
+86133|中國電信
+861340|中國移動
+861341|中國移動
+861342|中國移動
+861343|中國移動
+861344|中國移動
+861345|中國移動
+861346|中國移動
+861347|中國移動
+861348|中國移動
+86135|中國移動
+86136|中國移動
+86137|中國移動
+86138|中國移動
+86139|中國移動
+86145|中國聯通
+86147|中國移動
+86150|中國移動
+86151|中國移動
+86152|中國移動
+86153|中國電信
+86155|中國聯通
+86156|中國聯通
+86157|中國移動
+86158|中國移動
+86159|中國移動
+86173|中國電信
+86176|中國聯通
+86177|中國電信
+86178|中國移動
+86180|中國電信
+86181|中國電信
+86182|中國移動
+86183|中國移動
+86184|中國移動
+86185|中國聯通
+86186|中國聯通
+86187|中國移動
+86188|中國移動
+86189|中國電信
+861703|中國移動
+861705|中國移動
+861706|中國移動
+86172|中國移動
```

Comment:
Technically the file should be sorted: this was an error in the English one. I'm fixing that, so I'm happy to fix yours at the same time: or you can fix the 86.txt files now if you feel like it.